service_container/factories.rst

```diff
@@ -100,6 +100,82 @@ create its object:
     the configured class name may be used by compiler passes and therefore
     should be set to a sensible value.
 
+You can also use the ``factory`` option to create services from static methods
+of the class itself.
+Let's suppose the ``NewsletterManager`` class has a ``create()`` method that needs
+to be called to create the object and need a send email address::
+
+    // src/Email\NewsletterManager.php
+    namespace App\Email;
+
+    // ...
+
+    class NewsletterManager
+    {
+        private string $sender;
+
+        public static function create(string $sender): self
+        {
+            $newsletterManager = new self();
+            $newsletterManager->sender = $sender;
+            // ...
+
+            return $newsletterManager;
+        }
+    }
+
+You can use the ``factory`` option to define which method of which class must be
+called to create its object. In this case we don't need a dedicated factory class anymore:
```

Comment:
You never need a dedicated classe to declare a static method so this sentence doesn't look super useful to me